topi/python/topi/cuda/depthwise_conv2d.py

```diff
@@ -186,3 +186,106 @@ def traverse(OP):
 
     traverse(outs[0].op)
     return s
+
+
+def schedule_depthwise_conv2d_backward_input_nhwc(outs):
+    """Schedule for depthwise_conv2d nhwc backward wrt input.
+
+    Parameters
+    ----------
+    outs: Array of Tensor
+        The computation graph description of depthwise_conv2d
+        in the format of an array of tensors.
+
+    Returns
+    -------
+    s: Schedule
+        The computation schedule for depthwise_conv2d nhwc.
+    """
+    outs = [outs] if isinstance(outs, tvm.tensor.Tensor) else outs
+    s = tvm.create_schedule([x.op for x in outs])
+
+    def _schedule(Padded_out_grad, In_grad):
+        s[Padded_out_grad].compute_inline()
+
+        block_x = tvm.thread_axis("blockIdx.x")
+        thread_x = tvm.thread_axis("threadIdx.x")
+        _, h, w, c = In_grad.op.axis
+
+        fused_wc = s[In_grad].fuse(w, c)
+
+        fused_hwc = s[In_grad].fuse(h, fused_wc)
+        xoc, xic = s[In_grad].split(fused_hwc, factor=128)
+
+        s[In_grad].bind(xoc, block_x)
+        s[In_grad].bind(xic, thread_x)
+
+    def traverse(OP):
```

Comment:
traverse(OP) is for operator fusion, but backward propragation won't do operator fusion, maybe it's not needed here? @tqchen

Reply:
I think it does not hurt to do such style in case there is fusion chance if we do things like relu conv fusion and gradient can be fused